src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap);
+
+* {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  display: flex;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  height: 60px;
+  padding: 0;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+  height: 60px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.link {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  cursor: pointer;
+  padding: 22px 0;
+}
+
+.link:hover {
+  color: #00acdc;
+}
+
+.link:focus {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+  top: 3px;
+}
+
+.is-active::after {
+  position: relative;
```

Comment:
```suggestion
  position: absolute;
```

Reply:
with position absolute line is disappeared

Reply:
with position absolute line is disappeared